The fortify header for wchar.h seems to assume that string.h is included
beforehand. Otherwise it fails with the following error:

	In file included from /usr/include/fortify/strings.h:22,
			 from /usr/include/string.h:57,
			 from /usr/include/fortify/wchar.h:30,
			 from /home/soeren/src/aports/community/radare2/src/radare2-5.0.0/libr/include/r_util/r_str.h:4,
			 from uleb128.c:3:
	/usr/include/fortify/string.h:144:1: error: 'mempcpy' undeclared here (not in a function); did you mean 'memccpy'?
	  144 | _FORTIFY_FN(mempcpy) void *mempcpy(void *__d, const void *__s, size_t __n)
	      | ^~~~~~~~~~~

Including string.h fixes the error.

diff -upr radare2-5.0.0.orig/libr/include/r_util/r_str.h radare2-5.0.0/libr/include/r_util/r_str.h
--- radare2-5.0.0.orig/libr/include/r_util/r_str.h	2020-12-22 12:42:23.032644219 +0100
+++ radare2-5.0.0/libr/include/r_util/r_str.h	2020-12-22 12:42:30.796011296 +0100
@@ -1,6 +1,7 @@
 #ifndef R_STR_H
 #define R_STR_H
 
+#include <string.h>
 #include <wchar.h>
 #include "r_str_util.h"
 #include "r_list.h"
